### PR TITLE
Initialize project for review

### PR DESCRIPTION
**Pull request by AjroudiRida**
## Initializing the project for the review process

in this project:
- copied the youtube clone project that I completed in this micro checkpoint https://microverse.notion.site/HTML-CSS-Get-a-head-start-275eb85fd34b4416aa06ec635d69cdaf

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+                <li><a href="#"><span class="material-symbols-outlined">
+                    notifications
+                    </span></a></li>
+                <li><a href="#"><span class="material-symbols-outlined">
+                    video_call
+                    </span></a></li>
+                <li><a href="#"><span class="material-symbols-outlined">
+                        account_circle
+                </span></a></li>
+            </ul>
```
Comment (AjroudiRida) [inline index.html:26]: Please in the header section use the semantic tag "nav" for the menu and wrap your unordered list inside the nav tag. the semantic tag nav can give a boost in ranking in Google, plus it helps screen readers to understand that it's a menu section.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+  grid-template-rows: repeate(4, 100px);
+  grid-auto-rows: 100px;
+  row-gap: 10px;
+}
+
+.messages {
+  display: flex;
+  align-items: center;
+  
+}
```
Comment (AjroudiRida) [inline style.css:127]: Please add linters to your repository to check CSS errors. Linters is a tool that will help you write a structured and clean CSS style which will improve clarity.

**Review by AjroudiRida — COMMENTED** [review]
Hi @AjroudiRida

Good work so far!
You've still got a few things to work on before you can move on to the next project, but you're almost there!

**Highlights**
✔️ Good PR
✔️ Good use of Grid CSS
✔️ Good use of Flexbox
✔️ The document is well structured

**Required Changes**

🙏Add linters to your repository
🙏Apply the readme template

Check the comments below for more details.

**Review by Strangeal — APPROVED** [review]
Hi @AjroudiRida, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Highlights
Correct use of inline comments ✅
Polite comments ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**